cocos/ui/UIListView.cpp

```diff
@@ -455,42 +455,33 @@ void ListView::setDirection(Direction dir)
     ScrollView::setDirection(dir);
 }
     
-void ListView::requestRefreshView()
+void ListView::refreshView()
 {
-    _refreshViewDirty = true;
+    forceDoLayout();
 }
 
-void ListView::refreshView()
+void ListView::requestDoLayout()
+{
+    _innerContainerDoLayoutDirty = true;
+}
+
+void ListView::doLayout()
```

Comment:
No. `Layout::doLayout()` is doing layout of its children. But `ListView` has only one child `innerContainer`. Actually `ListView` is not a `Layout`, but the `innerContainer` is. So `innerContainer->forceDoLayout()` is called below.